internal/storage/runtime.go

```diff
@@ -256,6 +256,7 @@ func (r *runtimeService) createContainerOrPodSandbox(systemContext *types.System
 	if err != nil {
 		return ContainerInfo{}, err
 	}
+	defer json.Pool(mdata)
```

Comment:
in this case it might make more sense to call `json.Pool(mdata)` right after the data is used, i.e. after line 273